app/controllers/conferences_controller.rb

```diff
@@ -12,16 +15,46 @@ def show
     @conference = Conference.find(params[:id])
   end
 
+  def create
+    @conference = Conference.new(conference_params)
+    generate_gid(@conference)
+    @team = Team.find(params[:team_id])
+
+    respond_to do |format|
+      if @conference.save
+        format.html { redirect_to params[:redirect_to].presence || edit_team_path(@team), notice: 'Conference was successfully created.' }
+        format.json { render action: :edit, status: :created, location: @team }
+      else
+        format.html { render action: :new }
+        format.json { render json: @conference.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
   def redirect
     redirect_to orga_conferences_path
   end
 
   private
+
+  def generate_gid(conference)
+    conference.gid = Time.now.getutc
```

Comment:
Nice idea! 
Maria asked for some way to distinguish when a conference was suggested by a team or not. However, the idea is that the team suggests - not the student. So maybe we can make this GID in this format:
"2017-#{Time.now.getutc.to_i}-teamID"

What do you think?